dist/resource-translator/index.js

```diff
@@ -584,7 +582,7 @@ function isURLSearchParams(val) {
  * @returns {String} The String freed of excess whitespace
  */
 function trim(str) {
-  return str.replace(/^\s*/, '').replace(/\s*$/, '');
+  return str.trim ? str.trim() : str.replace(/^\s+|\s+$/g, '');
```

Comment:
I'm curious do you hand edit the _dist/resource-translator/index.js_ file? That is generated from the build step.

Reply:
That is strange, nope I did not touch that manually. I just did a build and committed that.